src/tasks/listener.py

```diff
@@ -58,7 +59,10 @@ async def _receive(self, message: str):
             position_data = data["data"]
             if position_data is None:
                 return
-
+            if self._settings.site.freeze != None and self._settings.site.freeze < (time.time() * 1000):
+                for team_data in position_data["positions"]:
+                    team_data['progress'] = 0
+                    team_data['speed'] = 0
```

Comment:
Will crash if `positions` is not in position_data.
We check if it exists just underneath